nav/support/attrdict.py

```
@@ -16,3 +17,10 @@ class AttrDict(dict[str, Any]):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
         super(AttrDict, self).__init__(*args, **kwargs)
         self.__dict__ = self
+
+    # This is a stupid thing to do, but it's necessary to avoid mypy from complaining
+    # about missing attributes. mypy ignores attributes for classes that have a
+    # __getattr__ method.
+    @no_type_check
+    def __getattr__(self, name: str) -> Any:
```

Ruff (0.14.3)

Dynamically typed expressions (typing.Any) are disallowed in `__getattr__`

(ANN401)